tests/foreman/ui/test_contentview.py

```diff
@@ -115,6 +115,19 @@ def setup_to_create_cv(self, repo_name=None, repo_url=None, repo_type=None,
         entity_mixins.TASK_TIMEOUT = old_task_timeout
         return repo_id
 
+    def _get_cv_version_environments(self, cv_version):
+        """
+        Return the list of environments promoted to the version of content
+        view.
+        The content view web page must be already opened
+        :param cv_version: The version of the current opened content view
+        :type cv_version: str
+        :rtype: list[str]
+        """
```

Comment:
```python
        """Return the list of environments promoted to the version of content
        view. The content view web page must be already opened.

        :param cv_version: The version of the current opened content view
        :type cv_version: str
        :rtype: list[str]

        """
```
Tip: you can use any online rst editor to check if your docstring is ok, for example [rst.ninjs.org](http://rst.ninjs.org/)